client/components/Pipeline/IndividualButton.tsx

```diff
@@ -10,14 +10,21 @@ const StyledButton = styled(Button)`
 `;
 
 interface DropdownProps {
-  loggedInPartnerName: string;
+  loggedInPartnerId: string;
+  reloadKanbanCompanies: (
+    currentTeam: any,
```

Comment:
all the reloadKanbanCompanies property parameter types (currentTeam, currentPartner) should be type string, not any